Orange/widgets/visualize/owtreeviewer.py

```diff
@@ -358,28 +358,26 @@ def update_node_info_cls(self, node):
         else:
             modus = np.argmax(distr)
             tabs = distr[modus]
-            text = self.domain.class_vars[0].values[int(modus)] + "<br/>"
+            text = f"{self.domain.class_vars[0].values[int(modus)]}<br/>"
         if tabs > 0.999:
-            text += "100%, {}/{}".format(total, total)
+            text += f"100%, {total}/{total}"
```

Comment:
PyLint complained about this line. The nicest solution was to use f-strings. Other changes in this function are for consistency -- as well as for readability, too.